scripts/integration-k.sh

```diff
@@ -18,7 +18,7 @@ if make --version | grep -q 'GNU Make 4' 2>/dev/null
 then
     MAKE="make --output-sync --jobs ${JOBS:?} --directory $TOP"
 else
-    MAKE="make -j ${JOBS:?} -C $TOP"
+    MAKE="make -C $TOP"
```

Comment:
Why remove the parallelism?

Reply:
Thank you for taking a look!

I didn't notice this, good catch. Unfortunately the commit message for this change is not descriptive at all, so I'll try to figure out why the original author of the PR decided to remove this.

Reply:
I reverted the change. There was one strange failure regarding the integration tests, but I couldn't reproduce it again on Jenkins or locally.